tasks/pruner.yml

```diff
@@ -25,9 +25,16 @@
     src: "openshift-rolebinding.v2.yml.j2"
     dest: "{{ osbs_openshift_home }}/{{ inventory_hostname }}-{{ osbs_namespace }}-rolebinding-{{ item.name }}.yml"
   with_items:
-  - name: build-pruner
+  - name: "{{ osbs_namespace }}-build-pruner-read"
+    role: "cluster-reader"
```

Comment:
> cluster-reader (to list BuildConfigs)

Can the `build-config-change-controller` really not list BCs?

Even so, surely there is a lesser role for the purpose, perhaps an ordinary project `RoleBinding` to the `view` role?

Reply:
I suspect Tim ran into issues when using `oc adm` that even when specifying a namespace it was still listing BuildConfigs outside of the namespace.

Reply:
It's actually required in order to list builds in the namespace. `build-config-change-controller` allows listing BuildConfigs and deleting Builds, but not listing Builds.

I've avoided the requirement for the 'cluster-reader' clusterrole now, and replaced it with the view role in the namespace.